README.md

```diff
@@ -8,7 +8,7 @@
 
 A native client to access Discourse server for Apple Platforms.
 
-![Menu Helper](Forumate/Other/Assets.xcassets/AppIcon.appiconset/icon_256x256@2x.png)
+![Menu Helper](Forumate/Other/Assets.xcassets/AppIcon.appiconset/AppIcon-macOS-1024.png)
```

Comment:
We'd better use a 256@2x/512 size image otherwise it will be too big to show in the README.(Or you can use some html trick to resize it)[Not recommended].

Reply:
Fixed in 4f9238a3ce9d81df831cc2358e35871eb7d0186e .